### ISSUE TITLE
Tried 2.5 on TrueNAS VM and backup and restore don't work. It just completely stalls ; no way to "go back": it's frozen

### ISSUE THREAD

**Issue by skirsch** [rescuezilla/rescuezilla#476]
I tried 2.5 in my Truenas Scale VM, and everything worked except backup and restore which both got to the very last step and there is a complete freeze of the app as soon as it starts a backup or restore.

Easy to replicate. Freezes every time.

Went back to 2.4 and the exact same configuration worked fine.

top showed nothing was running.

Happy to have you look into on my machine if you can't reproduce as I can open up the SPICE viewer port for you.

**Comment by shasheene** [comment]
This is on the default 'Noble' release?

It would be interesting to see the log file when running `/usr/sbin/rescuezilla` from a Terminal (after closing the regular Rescuezilla window).

**Comment by skirsch** [comment]
No, I'm running the image on the latest TrueNAS server as it's own standalone VM, basically booting it from the 2.4 CD.

So things work fine for 2.4, but when I switch to boot 2.5 as the VM, that's when the problem happens

But it's just the when you start a backup or restore after configuring the GUI, so I can tail -f a logfile if you tell me the name of what to tail :)
